syftlib/src/main/java/org/openmined/syft/Syft.kt

```diff
@@ -19,6 +19,15 @@ import java.util.concurrent.atomic.AtomicBoolean
 
 private const val TAG = "Syft"
 
+/**
+ * The Syft Class is the main worker handling all the federated classes. 
+ * You are only required a single instance of the syft worker running in Android.
+ * @property syftConfig is  for the adjustable properties of the syft worker.
+ * @property deviceMonitor is the job hosting this plan.
+ * @property authToken is to request to an authentication token.
```

Comment:
authToken is not the request rather it is the third party JWT authentication token